src/amdgpu.cpp

```diff
@@ -78,8 +78,15 @@ void AMDGPU::get_instant_metrics(struct amdgpu_common_metrics *metrics) {
 		metrics->current_uclk_mhz = amdgpu_metrics->current_uclk;
 
 		metrics->gpu_temp_c = amdgpu_metrics->temperature_edge;
-		indep_throttle_status = amdgpu_metrics->indep_throttle_status;
 		metrics->fan_speed = amdgpu_metrics->current_fan_speed;
+
+		indep_throttle_status = amdgpu_metrics->indep_throttle_status;
+		bool temp_hotspot_flag = ((indep_throttle_status >> TEMP_HOTSPOT_BIT) & 1) == 1;
+		// ThrottlingPercentage for TEMP_HOTSPOT on RDNA 3 dGPU is almost always greater then or equal to 1.
+		// So the AMDGPU driver set the TEMP_HOTSPOT bit even in idle state.
+		if (temp_hotspot_flag && amdgpu_metrics->temperature_hotspot < 90) {
```

Comment:
We don't *know* if it's actually throttling, so should we say it's throttling just because it's above 90c?
Junction can easily reach about 100c without throttling, I'm pretty sure.
Maybe just ignore this case until it's fixed?

Reply:
Yeah I think so, just clear it until we can use it properly

Reply:
Done. https://github.com/flightlessmango/MangoHud/pull/1690/commits/5d2ec01b4776dec783dc08974d51bb3bd2bdc516